h2o-algos/src/main/java/hex/coxph/CoxPHUtils.java

```diff
@@ -4,13 +4,6 @@
 
 class CoxPHUtils {
 
-  static double[][] malloc2DArray(final int d1, final int d2) {
-    final double[][] array = new double[d1][];
-    for (int j = 0; j < d1; ++j)
-      array[j] = MemoryManager.malloc8d(d2);
-    return array;
-  }
-
   static double[][][] malloc3DArray(final int d1, final int d2, final int d3) {
```

Comment:
@michalkurka maybe we can move this 3D method to MemoryManager as well? or you think it is not that common  to put it there?